korean/explore-outside/investigate-marshmallow-structure/approach-green-people/approached-green-people.md

```diff
@@ -1,4 +1,6 @@
 초록 인간들에게 다가가자 그들이 고개를 돌려 당신이 있는쪽을 바라보고 있어요.
 당신은 어떻게 할까요?
 
-[마시멜로 방 밖으로 되돌아간다] (../../explore-outside.md)
\ No newline at end of file
+[마시멜로 방 밖으로 되돌아간다] (../../explore-outside.md)
+
+[초록 인간들에게 더 가까이 가본다] (./getting-closer.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.